models/Property.php

```diff
@@ -129,6 +129,8 @@ public function getCid()
     }
 
     /**
+     * enum('document','asset','object')
+     *
      * @return string
```

Comment:
What about using PHPStan's type annotations for this? [Like this:](https://phpstan.org/writing-php-code/phpdoc-types#literals-and-constants)
```suggestion
     * @return 'document'|'asset'|'object'
```

Or maybe even [with an alias](https://phpstan.org/writing-php-code/phpdoc-types#local-type-aliases), which can be used for the property and its getters/setters.

Reply:
@jdreesen This needs more refactoring so not the priority for us. we would be happy to receive a PR for this improvement. thanks for your suggestion 😉

Reply:
I'm afraid I won't have time for that in the near future :/